server/app/modules/hunts/router.py

```diff
@@ -26,4 +27,15 @@ async def load_past_hunts():
   return HuntResponseModel(
     message="fetched hunts",
     content=result
+  )
+
+@router.post(
+  "/join_hunt",
+  status_code=200,
+  response_model=HuntResponseModel
+)
+async def load_upcoming_hunts():
+  result = await service.get_upcoming()
```

Comment:
Why is this function called load_upcoming_hunts under the join hunt endpoint? This function should be called `join_hunt` or something like that, and if it doesn't have functionality yet make sure to add a comment indicating that its just an empty method for now, and add a default return or something.